rllib/connectors/env_to_module/mean_std_filter.py

```diff
@@ -116,9 +116,16 @@ def __call__(
         # anymore to the original observations).
         for sa_episode in self.single_agent_episode_iterator(episodes):
             sa_obs = sa_episode.get_observations(indices=-1)
-            normalized_sa_obs = self._filters[sa_episode.agent_id](
-                sa_obs, update=self._update_stats
-            )
+            try:
```

Comment:
Make the error better, that shows up when `multi_agent=True` c'tor arg is forgotten.